libcodechecker/server/server.py

```diff
@@ -43,8 +43,8 @@
 from libcodechecker import session_manager
 from libcodechecker.logger import LoggerFactory
 from libcodechecker.util import get_tmp_dir_hash
+from libcodechecker.server import database
```

Comment:
If we are importing the `server.database` module, let's not also import `server.database.database` as `database` in line 55. Didn't it cause name collisions?